run-tests.py

```diff
@@ -187,15 +187,16 @@ def test_notebook(path, executable="python"):
     print("Test " + path + " ... ", end="")
     sys.stdout.flush()
 
-    # Make sure the notebook has a "%pip install pybamm -q" command, for using Google
-    # Colab
+    # Make sure the notebook has a "%pip install pybamm[plot,cite] -q" command, for using Google Colab # noqa: E501
```

Comment:
I am nitpicking, but we should split this comment into 2 lines and get rid of the `noqa` comment.

Reply:
(ruff's purpose is to let us know that we need to split up our long lines. Adding `noqa` to a line that can be split up defeats the purpose.)

Reply:
Will keep this in mind 

> (ruff's purpose is to let us know that we need to split up our long lines. Adding `noqa` to a line that can be split up defeats the purpose.)